game-core/src/main/java/games/strategy/triplea/attachments/PoliticalActionAttachment.java

```diff
@@ -153,4 +178,17 @@ public void validate(final GameData data) throws GameParseException {
                 this::resetRelationshipChange))
         .build();
   }
+
+  /**
+   * A relationship change specified in a political action attachment. Specifies the relationship type that will exist
+   * between two players after the action is successful.
+   */
+  @AllArgsConstructor(access = AccessLevel.PACKAGE)
+  @EqualsAndHashCode
+  @Immutable
+  public static final class RelationshipChange {
+    public final String player1Name;
+    public final String player2Name;
+    public final String relationshipTypeName;
```

Comment:
Initial comments touch on it already but essentially boils down to whether its worth making these types into PlayerID and RelationshipType objects instead of the raw Strings.

Reply:
The latest commits implement your suggestions.  I think they worked out well, but please take another look.

The only questionable change is the new `PoliticalActionAttachment#getRelationshipChanges()` method.  It only differs from the existing `getRelationshipChange()` method by the last character.

I re-ran the manual test outlined above to sanity check the latest changes.